docs/content/latest/legal/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Legal information
+headerTitle: Legal information
+linkTitle: Legal
+description: Learn about Yugabyte licenses and third party software.
+image: /images/section_icons/index/quick_start.png
+headcontent: Learn about YugabyteDB licenses and third party software.
+type: page
+section: MISC
+menu:
+  latest:
+    identifier: legal
+    weight: 3000
+---
+
+## Licenses
+
+Source code in the [YugabyteDB repository](https://github.com/yugabyte/yugabyte-db/) is variously licensed under the Apache License 2.0 and the Polyform Free Trial License 1.0.0. A copy of each license can be found in the [licenses](https://github.com/yugabyte/yugabyte-db/tree/master/licenses) directory.
+
+The build produces two sets of binaries:
+
+- The entire database with all its features (including the enterprise ones) are licensed under the [Apache License 2.0](./apache-license).
+- The binaries that contain `-managed` in the artifact and help run a managed service are licensed under the [Polyform Free Trial License 1.0.0](./polyform-license).
+
+By default, the build options generate only the Apache License 2.0 binaries.
+
+## Individual contributor license agreement (CLA)
+
+As an open source project with a strong focus on the user community, Yugabyte welcomes contributions as GitHub pull requests. Before someone can contribute to the YugabyteDB open source project, the online form [Yugabyte Individual Contributor License Agreement](https://docs.google.com/forms/d/11hn-vBGhOZRunclC3NKmSX1cvQVrU--r0ldDLqasRIo/viewform?edit_requested=true) must be completed. To see a copy of the complete agreement, go to [Yugabyte contributor license agreement (CLA)](./cla).
```

Comment:
link should point to https://cla-assistant.io/yugabyte/yugabyte-db

Reply:
done